lib/origen_testers/origen_ext/generator/flow.rb

```diff
@@ -12,7 +12,8 @@ class Flow
       # of the ATP AST originated from
       def create(options = {}, &block)
         options = {
-          reload_target: true
+          reload_target: true,
+          name:          OrigenTesters::Flow.name_stack.empty? ? nil : OrigenTesters::Flow.name_stack.last
```

Comment:
I don't think we need this empty? check anymore, maybe we should just pop the stack here.